src/components/m-button/index.scss

```diff
@@ -79,14 +79,16 @@
   &[aria-disabled] {
     .m-button__icon {
       /* stylelint-disable declaration-no-important */
-      color: $color-prim-gray-silver !important;
-      border-left-color: $color-prim-gray-silver !important;
+      color: $color-button-icon--disabled !important;
```

Comment:
hmh, where are those variables defined?

Reply:
Okay they come from the mixins file, this is the wrong place. All vars come from settings file.